validator/src/main/resources/validations/otelFargate-metric-validation.yml

```diff
@@ -0,0 +1,4 @@
+-
```

Comment:
Fix the file name. It should be lower kebab case.

Reply:
Also, add the Fargate Keyword. All name should be consistent like `eks-fargate-container-insights-something`.

Reply:
we can use eks-fargate-cwci instead of eks-fargate-container-insights- since container insights is long

Reply:
sounds good.